src/main/java/Duke.java

```diff
@@ -34,28 +32,50 @@ static void printFarewellMessage() {
         String farewell = "Bye. Hope you will complete everything for today!";
         System.out.println(farewell);
     }
-    static void recordTask(String phrase) {
-        System.out.println("Alright. I'll put it on the list.");
+
+    static void printPutTaskOnList() {
+        String toPrint = "Alright. I'll put it on the list.";
+        System.out.println(toPrint);
+    }
+
+    static void printNumberOfTasks() {
+        if (Timetable.size() <= 1) {
+            System.out.println("For now, you have " + Timetable.size() + " task on the list");
+        } else {
+            System.out.println("For now, you have " + Timetable.size() + " tasks on the list");
+        }
+    }
+
+    static int parseDoneInputToInt(String phrase) {
+        phrase = phrase.replace("done ", "");
+        return Integer.parseInt(phrase);
+    }
+
+    static Timetable parsePhraseToTask(String phrase) {
+        Timetable task;
 
         if (phrase.contains("todo")) {
-            Todo todo = new Todo(phrase.replace("todo ", ""));
-            Timetable.addTasks(todo);
+            task = new Todo(phrase.replace("todo ", ""));
         } else if (phrase.contains("deadline")) {
             phrase = phrase.replace("deadline ","");
             String[] whatAndWhen = phrase.split(" /by ");
```

Comment:
whatAndWhen is not very descriptive, perhaps use something along the lines of taskAndTimeline?